lablueprint/skid-row-arts-alliance
# Event & Artwork Screen Saving

### Completed
- Toggle to add/remove event in database based on eventID
- Determine if event has been saved or not (and set toggle accordingly)
- Toggle to add/remove if artwork in database based on artworkID
- Determine if artwork has been saved or not (and set toggle accordingly)

### Unfinished
- UserID for both event and artwork saving is currently hardcoded
- Save state to redux

### Testing
For both event and artwork cards...
- Enter card and toggle switch on
- Ensure corresponding ID was added to User 63e33e2f578ad1d80bd2a347 savedEvent/Artwork array 
- Return to gallery/event screen
- Re-enter card
- Ensure save toggle is still on
- Toggle off
- Ensure corresponding ID was removed form User 63e33e2f578ad1d80bd2a347  savedEvent/Artwork array
- Return to gallery/event screen
- Re-enter card
- Ensure save toggle is still off
